modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -18952,6 +18955,7 @@ uninstalled=Uninstalled
 uninvite=Uninvite
 unique=Unique
 unique-key-for-referencing-the-object-definition=Unique Key for Referencing the Object Definition
+unique-key-for-referencing-the-object-folder=Unique Key for Referencing the Object Folder
```

Comment:
We can create a more generic language key for this case, such as "unique-key-for-referencing-the-{x}," and create more reusable language keys like "Object Folder," "Object Definition," and "Picklist Definition."

Reply:
Perfect! I'll do it 👍🏻